geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ServiceVersionNotFoundException.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.internal.cache.tier.sockets;
```

Comment:
I'm not convinced we need to add another class to this already overflowing package. I feel that this is really a client protocol concept. As it is currently the only code that cares about ServiceVersions. Maybe this exception is moved into the `org.apache.geode.internal.cache.client.protocol` package?

Reply:
Makes sense, will move this and the other exception (ServiceLoadingFailureException).